lib/host_kafka.py

```diff
@@ -0,0 +1,33 @@
+import socket
+from contextlib import closing
+
+from app.config import Config
+from app.environment import RuntimeEnvironment
+from app.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def _get_bootstrap_servers(kafka_socket, servers):
```

Comment:
nice compression - new the name 
```suggestion
def _any_bootstrap_server_connects(kafka_socket, servers) ->  bool:
```

seems like match